app/views/policies/_form.html.slim

```diff
@@ -0,0 +1,28 @@
+.x_content
+  /! Smart Wizard
+  p Policy details
+  #wizard.form_wizard.wizard_horizontal
+    ul.wizard_steps.anchor
+      li
+        a.selected href="#step-1" isdone="1" rel="1" 
+          span.step_no 1
+          span.step_descr
+            | Step 1
+            br/
+            small Step 1 Policy Details
+      li
+        a.disabled href="#step-2" isdone="0" rel="2" 
+          span.step_no 2
+          span.step_descr
+            | Step 2
+            br/
+            small Step 2 Insurance Details
+      li
+        a.disabled href="#step-3" isdone="0" rel="3" 
+          span.step_no 3
+          span.step_descr
+            | Step 3
+            br/
+            small Step 3 Nominy Detail
```

Comment:
@KaranBamaniya-BTC : It should display Personal detail of policy holder ,create a new tab for address